docker/key-manager/run.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# Copyright (c) The Libra Core Contributors
+# SPDX-License-Identifier: Apache-2.0
+set -ex
+
+image="${1:-libra_key_manager:latest}"
+
+docker network create --subnet 172.18.0.0/24 testnet || true
+
+docker run \
+    -e JSON_RPC_ENDPOINT="http://127.0.0.1:8080" \
```

Comment:
@mgorven @davidiw what is the best way to define these? Right now the script just uses random values (e.g., localhost:8080 and a garbage token), but this is obviously not right. Do we have a story for how we'll know these constants at deployment time?

Reply:
The values in here are designed for running locally.

Reply:
e.g., the json rpc interface exposed by each validator node can be accessed at localhost (assuming) the KM and node are deployed on the same machine, but the port is "randomized", so we don't really know this until the node config exists... similarly, what are our assumptions for vault?

Reply:
For running docker containers locally we assume fixed port numbers. `run.sh` isn't used anywhere else, Terraform/Kubernetes handle this in that case.

Reply:
Look at docker/mint/run.sh 
let's be consistent like that...